static/defaults.config.php

```diff
@@ -614,6 +614,11 @@
 		// Timeout in seconds for fetching the XRD links and other requests with an expected shorter timeout
 		'xrd_timeout' => 20,
 	],
+	'performance' => [
+		// max_response_data_size (Integer)
+		// Maximum allowed response data size in Bytes, default is hard-coded value from code
```

Comment:
```suggestion
		// Maximum allowed outgoing HTTP request response data size in Bytes. Does not affect incoming requests to this node.
		// Warning: Lowering this value can help with some PHP memory exhaustion issues, but can also partially break some federation features e.g. large posts may not be fetched or received from remote servers.
```

Reply:
Done.

Reply:
Please add the second line with the Warning as well.

Reply:
This suggested change isn't signed so I didn't use the tool here directly. Then it was marked bold.